datalake-spark31/src/test/scala/bio/ferlab/datalake/spark3/transformation/TransformationSpec.scala

```diff
@@ -99,6 +100,21 @@ class TransformationSpec extends AnyFlatSpec with GivenWhenThen with Matchers {
       )
   }
 
+  val testData = Seq(
+    ("universinformationnel", "456123789")
```

Comment:
this shouldbe inside the test, just above 'val job...'

Reply:
done